### PR TITLE
updated skeleton loader doc

### PR DESCRIPTION
**Pull request by RitupL**

### CODE REVIEW

File: learn/react-native/app-loader.md [hunk 1]
```diff
@@ -47,17 +72,23 @@ In the following example, apply child Widgets of `widgetName` to show the Skelet
 
 #### Change Skeleton Colors
 
-You can control Skeleton Loader colors using styles. There are three style properties as following: 
+You can customise the color, width, and other properties of Skeleton Loader . These changes can be applied at three levels:
 
-1. skeleton
-2. skeleton-gradient
-3. skeleton-gradient-foreground
+- Page level
```
Comment (swethakundaram) [inline learn/react-native/app-loader.md:77]: point to the section where it applies for page, widget, and class headers.

File: learn/react-native/app-loader.md [hunk 2]
```diff
@@ -5,38 +5,63 @@ sidebar_label: "App Loader"
...
+
+### Enabling Skeleton Loader
+
+Skeleton Loader can be enabled when creating an application. To apply the Skeleton Loader, go to the **Project Settings** dialog and choose **Skeleton Loader** as the Application Loader.
+
+<iframe width="560" height="315" src="https://embed.app.guidde.com/playbooks/4tzkn2vjCNnJUZaXi5xYUK"   frameborder="0" allow="autoplay; encrypted-media" allowfullscreen="allowfullscreen"></iframe>
+
+:::note
+
+Skeleton Loader gets applied across the application only if a service variable is available for the application and the Spinner Context is also set as Page.
```
Comment (swethakundaram) [inline learn/react-native/app-loader.md:52]: point to service variable documentation and spinner context documentation.

File: learn/react-native/app-loader.md [hunk 3]
```diff
@@ -47,17 +72,23 @@ In the following example, apply child Widgets of `widgetName` to show the Skelet
...
-1. skeleton
-2. skeleton-gradient
-3. skeleton-gradient-foreground
+- Page level
+- Widget level
+- Class level
+
+During Skeleton customization, Skeleton Loader is categorized into three classes where,
+
+1. skeleton: It is the outer container of the Skeleton component.
```
Comment (swethakundaram) [inline learn/react-native/app-loader.md:83]: Use apt formatting, such as code blocks for classes in this case. And use a header for this section.

File: learn/react-native/app-loader.md [hunk 4]
```diff
@@ -112,25 +143,79 @@ className: headerLabel
 }
 ```
 
+#### Skeleton Loader in Container Type Widgets
+
+Container type widgets are recently getting supported by Skeleton Loader. This category includes the below widgets,
```
Comment (swethakundaram) [inline learn/react-native/app-loader.md:148]: remove the word recently and add a better explanation for this.

File: learn/react-native/app-loader.md [hunk 5]
```diff
@@ -112,25 +143,79 @@ className: headerLabel
...
+- Grid Layout
+- Flex Layout
+- Accordion
+- Tabs
+- Wizard
+- Panel
+- Container
+- Tile
+
+The widgets in this category have child widgets on which the Skeleton Loader is not applied by default. To apply it on the child widgets within the Container type widget, set **`showskeletonchildren`** property as `true` in Markup script.
```
Comment (swethakundaram) [inline learn/react-native/app-loader.md:159]: please include the code snippet of the usage.

File: learn/react-native/app-loader.md [hunk 6]
```diff
@@ -112,25 +143,79 @@ className: headerLabel
...
+
+#### Customization using Lottie Animation
```
Comment (swethakundaram) [inline learn/react-native/app-loader.md:161]: Need code snippet and instructions to apply clearly. Use images and video in this case.

File: learn/react-native/app-loader.md [hunk 7]
```diff
@@ -112,25 +143,79 @@ className: headerLabel
...
+
+In Page, Prefab and Partial, the user can override the default skeleton appearance using Lottie animation. The **`animationResource`** property can be used in Markup script of Page, Prefab and Partial to provide the location of the Json file as Lottie animation resource file. The location of json file given as an resource, renders the animation through application's Lottie widget.
+
+:::note
+It is required to use Lottie widget in the application to implement the above.
```
Comment (swethakundaram) [inline learn/react-native/app-loader.md:166]: point to Lottie widget documentation

**Review by swethakundaram — CHANGES_REQUESTED** [review]

**Review by swethakundaram-wm — APPROVED** [review]

**Review by swethakundaram — APPROVED** [review]

**Pull request merged**